mbed_greentea/mbed_greentea_cli.py

```diff
@@ -401,9 +401,12 @@ def run_test_thread(test_result_queue, test_queue, opts, mut, build, build_path,
                                          verbose=verbose)
 
         # Some error in htrun, abort test execution
-        if host_test_result < 0:
+        if isinstance(host_test_result, int):
```

Comment:
Ah ok, thanks for clarifying. Yeah if you wanted to add a verbose-only log here that probably wouldn't hurt.